apps/documentation/src/routes/documentation/installation.mdx

```diff
@@ -13,25 +13,25 @@
 
 ## Requirements
 
-`Tuono` is a development environment built in rust and typescript that outputs a website written with both the
-languages. Since that you need the following tools installed on your computer to work with it:
+`Tuono` is a development environment built in rust and typescript that outputs a website written in both the
+languages. Since then, you need the following tools installed on your computer to work with it:
 
-- `Rust` - The rust programming language toolchain (Go [here](https://rustup.rs/) for installing both `rust` and `cargo`)
+- `Rust` - The Rust programming language tool chain (Go [here](https://rustup.rs/) for installing both `rust` and `cargo`)
 - `Cargo` - The rust package manager
-- `NodeJs` - The Javascript runtime (Go [here](https://nodejs.org/en/download/package-manager) for installing both `nodejs` and `npm`)
-- `npm` or `yarn` or `pnpm` - A javascript package manager
+- `NodeJs` - The JavaScript runtime (Go [here](https://nodejs.org/en/download/package-manager) for installing both `nodejs` and `npm`)
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The official spelling of this programming framework is “Node.js”. (NODE_JS[3])
Suggestions: `Node.js`
Rule: https://community.languagetool.org/rule/show/NODE_JS?lang=en-US&subId=3
Category: COMPOUNDING
<!-- __reviewdog__:ChA0N2FlZTFlNzEzYTliYzA2EgxMYW5ndWFnZVRvb2w= -->